### PR TITLE
OEL-257: Add Search API Autocomplete to Search View

### PR DESCRIPTION
**Pull request by GilNovacomm**

### CODE REVIEW

File: runner.yml.dist [hunk 1]
```diff
@@ -22,6 +22,7 @@ drupal:
     - "./vendor/bin/drush -y config-set system.performance css.preprocess 0"
     - "./vendor/bin/drush -y config-set system.performance js.preprocess 0"
     # Enable the modules.
+    - "./vendor/bin/drush en oe_showcase_default_content -y"
```
Comment (cmarcmar) [inline runner.yml.dist]: Shouldn't we enable modules in the config/sync/core.extension.yml file instead?

Comment (drishu) [reply runner.yml.dist]: in profiles we should not export that

**Review by Maxfire — DISMISSED** [review]
reviewed and works fine.

**Review by drishu — APPROVED** [review]

**Review by Maxfire — APPROVED** [review]
approved :+1:

**Pull request merged**